src/Writer/StandaloneExtensionManager.php

```diff
@@ -83,8 +85,10 @@ public function add($name, $class)
      * Remove an extension.
      *
      * @param string $name
+     *
+     * @return void
      */
-    public function remove($name)
+    public function remove($name): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.